build/verify-features.ts

```diff
@@ -67,8 +67,21 @@ function findError(filename: string): string | void {
 		isHighlightedFeature = true;
 	}
 
-	const featureRegex = findFeatureRegex(featureId);
 	const lines = [];
+	let highlightedFeatureMatch;
+	do {
+		highlightedFeatureMatch = highlightedFeatureRegex.exec(readmeContent);
+		if (highlightedFeatureMatch) {
+			lines.push(readmeContent.slice(0, highlightedFeatureMatch.index).split(/\r?\n/).length);
+		}
+	} while (highlightedFeatureMatch);
+
+	if (lines.length > 0) {
+		return `ERR: ${featureId} should be described only once in the readme, but it is also described on line(s) ${lines.join(', ')}`;
+	}
```

Comment:
For some reason this regex returns matches only if the feature has been described more than once in the highlights section... not sure why that is happening

Reply:
The other regex (for the list of features) is matching all the descriptions